pyproject.toml

```diff
@@ -61,9 +61,9 @@ dependencies = [
     # pyproj 3.3+ required for Python 3.10 wheels
     "pyproj>=3.3",
     # rasterio 1.3+ required for Python 3.10 wheels
-    # rasterio 1.4+ no longer supports merging WarpedVRT objects
+    # rasterio 1.4.0-1.4.2 lack support for merging WarpedVRT objects
     # https://github.com/rasterio/rasterio/issues/3196
-    "rasterio>=1.3,<1.4",
+    "rasterio>=1.3,!=1.4.0,!=1.4.1,!=1.4.2",
```

Comment:
Unfortunately there's no cleaner way of doing this: https://stackoverflow.com/questions/29499542/exclude-certain-dependency-version-ranges-in-setuptools-pip